tests/integration/kubernetes/lib.sh

```diff
@@ -191,6 +191,15 @@ set_metadata_annotation() {
 	# yq set annotations in yaml. Quoting the key because it can have
 	# dots.
 	yq -i ".${annotation_key} = \"${value}\"" "${yaml}"
+
+	if [[ "${key}" =~ kernel_params ]] && [[ "${KATA_HYPERVISOR}" == "qemu-se" ]]; then
```

Comment:
IIUC the first check will have the test to use some pre-existing image instead of repacking one. If this is the expected behavior, it is an definitely an improvement :smiley:

Reply:
Yeah, we have an image `kata-containers-se.img` deployed in a designated place (i.e. `/opt/kata/share/kata-containers`) It is used unless a kernel parameter is changed. Thanks!